scripts/Diffraction/isis_powder/osiris_routines/osiris_algs.py

```diff
@@ -95,19 +97,18 @@ def set_empty(self, ws_name):
         return
 
     def calibrate_vanadium(self, calibration_file=""):
```

Comment:
Please can you rename this function to `calibrate_and_focus_vanadium` as well?

Reply:
ok